app/src/main/res/layout/activity_main.xml

```diff
@@ -22,11 +22,12 @@
         android:gravity="center"
         android:orientation="vertical">
 
-        <EditText
+        <AutoCompleteTextView
```

Comment:
I replaced the EditText with AutoCompleteTextView because it is something I think you guys should look into doing in the future since it helps with eliminating user input and makes it easier and more efficient for the user.
https://developer.android.com/guide/topics/text/autofill-optimize.